lib/jbuild

```diff
@@ -0,0 +1,31 @@
+(* -*- tuareg -*- *)
+#require "unix"
+
+let flags pkgs =
+  let cmd = "ocamlfind ocamlc -verbose" ^ (
+    List.fold_left (fun acc pkg -> acc ^ " -package " ^ pkg) "" pkgs
+  ) in
+  let ic = Unix.open_process_in
+    (cmd ^ " | grep -oEe '-ppx \"[^\"]+\"'")
```

Comment:
I guess it will be a reason more to make this library upgrade (finally!). But I am happy to live with it in the meanwhile